extensions/wikia/AdEngine/js/tracking/adInfoListener.js

```diff
@@ -51,7 +51,7 @@ define('ext.wikia.adEngine.tracking.adInfoListener',  [
 	}
 
 	function trackSlot(slot, status, adInfo) {
-		var slotFirstChildData = slot.container.firstChild.dataset,
+		var slotFirstChildData = slot.container ? slot.container.firstChild.dataset : {},
```

Comment:
This is the weak side of this solution. We need to hack all places in adengine component where adSlot object may be used and be prepared for a case when adSlot is notReallyAdSlot. And in future we'll have to be careful about this case.